rviz_common/include/rviz_common/ros_topic_display.hpp

```diff
@@ -206,11 +206,23 @@ class RosTopicDisplay : public _RosTopicDisplay
 
     try {
       // TODO(anhosi,wjwwood): replace with abstraction for subscriptions once available
+      rclcpp::SubscriptionOptions sub_opts;
+      sub_opts.event_callbacks.message_lost_callback =
+        [&](rclcpp::QOSMessageLostInfo & info)
+        {
+          std::stringstream sstm;
+          sstm << "Some messages were lost:\n>\tNumber of new lost messages: " <<
```

Comment:
it would be good to include the topic name in the error message

Reply:
This will show in the UI next to the topic